README.md

```diff
@@ -1,8 +1,8 @@
-# EigenDA Proxy
+# EigenDA Proxy <!-- omit from toc -->
 
-A basic REST server for Rollups with the following endpoints:
-- POST routes: used by sequencers to encode their payloads (batches of transactions or state transitions) into EigenDA blobs, submit them to the EigenDA disperser to make them available, and receive a DA certificate to be submitted to the rollup's batcher inbox.
-- GET routes: used by validators to retrieve the blob from the EigenDA network given a DA Certificate, decode the blob back into the original payload, and return it to the validator, to be used in its derivation pipeline.
+A basic REST proxy server to interact with the EigenDA network:
+- POST routes: submit payloads (rollup txs, state-diffs, or anything really) that will be encoded into EigenDA blobs and submitted to the EigenDA disperser to make available for 2 weeks. A DA certificate of availability will be returned, which can be used to validate the availability and query the payload back.
```

Comment:
nit: these super long lines make it hard to read the markdown in a text editor or IDE without autowrapping. IMO it makes sense to manually insert line breaks at reasonable intervals. But as we've discussed before, this should probably be handled by the linter

Reply:
[fdacb98](https://github.com/Layr-Labs/eigenda-proxy/pull/330/commits/fdacb98f16fbf54bf3bdb2048a6f8a446ea404c1)